lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php

```diff
@@ -55,6 +57,18 @@ public function __construct(Traversable $iterator)
         $this->storeCurrentItem();
     }
 
+    /**
+     * @see https://php.net/countable.count
+     * @return int
+     */
+    #[ReturnTypeWillChange]
+    public function count()
```

Comment:
```suggestion
    public function count(): int
```

This will be perfectly fine on its own :)

Reply:
no problem :wink: